lib/TokenClient.php

```diff
@@ -64,7 +66,7 @@ public function __construct($channel, $tokenCluster, $cryptoEngineFactory)
      * @param Alias $alias to check
      * @return bool {@code true} if alias exists, {@code false} otherwise
      */
-    public function isAliasExists($alias)
+    public function resolveAlias($alias)
```

Comment:
don't make breaking changes please, mark the existing one as deprecated, point to the new one in the description